.github/workflows/mac-os.yml

```diff
@@ -14,14 +14,21 @@ jobs:
       # Clones a single commit from the libtock-rs repository. The commit cloned
       # is a merge commit between the PR's target branch and the PR's source.
       - name: Clone repository
-        uses: actions/checkout@v2.3.0
+        uses: actions/checkout@v2
+
+      # The brew commands are spread out over multiple steps so we can see the
+      # timings for each step in GitHub Actions' dashboard.
+      - name: Brew tap ARMmbed
+        run: brew tap ARMmbed/homebrew-formulae
+
+      - name: Brew update
+        run: brew update
+
+      - name: Brew install
+        run: brew install arm-none-eabi-gcc
```

Comment:
This looks fine but if you want an easy way to speed this up even more and enable version pinning, you can use this github action: https://github.com/fiam/arm-none-eabi-gcc . It works on mac and linux for the arm toolchain! 

I use it in my PR for switching the libtock-c CI to github actions (and in retrospect I probably should have mentioned it on the call today, sorry about that) https://github.com/tock/libtock-c/pull/207/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR25-R27

Reply:
I switch it to use that, and it saved about a minute, so I'll keep it. Thanks for pointing it out!